examples/OneOpenAir/OneOpenAir.ino

```diff
@@ -529,6 +549,11 @@ static void displayExecuteOta(OtaState state, String msg, int processing) {
     break;
   }
   case OtaState::OTA_STATE_SUCCESS: {
+    if (ag->isOne()) {
+      oledDisplay.showNewFirmwareUpdating(String(100));
```

Comment:
Not related to the PR but I would expect that we can write this here in this context

oledDisplay.showFirmwareUpdateProgress(100);

Reply:
It's relate to PR. Because firmware process has finish 100% but display sometime do not show 100%.

Reply:
Ehm, I mean that we should pass the value along as an int (and not a String)